usaspending_api/etl/management/commands/reveal_last_period_award_updates.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from usaspending_api.broker.helpers.last_load_date import get_last_load_date, update_last_load_date
+from usaspending_api.etl.management.helpers.recent_periods import retrieve_recent_periods
+
+logger = logging.getLogger("script")
+
+REVEAL_AWARD_SQL = """
+UPDATE awards
+SET update_date = now()
+WHERE id IN (
+    SELECT DISTINCT faba.award_id
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc ON
+        defc.group_name = 'covid_19'
+        AND faba.disaster_emergency_fund_code = defc.code
+    INNER JOIN submission_attributes sa ON
+        faba.submission_id = sa.submission_id
+    INNER JOIN dabs_submission_window_schedule dabs ON
+        dabs.id = sa.submission_window_id
+        AND dabs.submission_fiscal_year = {year}
+        AND dabs.submission_fiscal_month = {month}
+        AND dabs.is_quarter = {is_quarter}
+    INNER JOIN awards a on faba.award_id = a.id
```

Comment:
Is more performant to use this inner join or adding faba.award_id is not null?

Reply:
You're right. Changing to `faba.award_id is not null` reduced the inner query from 20s to 6s in QAT